esp-hal/src/uart.rs

```diff
@@ -315,13 +315,17 @@ impl Default for RxConfig {
 pub struct TxConfig {
     /// Threshold level at which the TX FIFO is considered empty.
     fifo_empty_threshold: u16,
+
+    /// Baud rate for the UART communication.
+    baudrate: u32,
```

Comment:
Baudrate is already in the global configuration, don't duplicate it here please.

Reply:
Fixed in [9d94956](https://github.com/esp-rs/esp-hal/pull/3177/commits/9d94956a51f2a8d5416ccca064224712bdc500f6)